packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerImage.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import { SwatchPicker, SwatchPickerOnSelectEventHandler, ImageSwatch } from '@fluentui/react-swatch-picker-preview';
+import seaImageSwatch from './assets/img-sea-swatch.jpg';
+import seaImageFile from './assets/img-sea.jpg';
+import bridgeImageSwatch from './assets/img-bridge-swatch.jpg';
+import bridgeImageFile from './assets/img-bridge.jpg';
+import parkImageSwatch from './assets/img-park-swatch.jpg';
+import parkImageFile from './assets/img-park.jpg';
```

Comment:
Please use CDN assets only in stories, this can't be exported to Stackblitz

Reply:
done